### PR TITLE
program to check if number can be expressed as sum of squares of two …

### PR DESCRIPTION
**Pull request by akhil-kk**
…whole numbers

program to check if number can be expressed as sum of squares of two whole numbers:

Input :  8

Output :
true

Input : 3

Output :
false

### CODE REVIEW

File: 014_number_as_sum_of_squares.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//program to check if number can be expressed as sum of squares of two whole numbers
+const numberAsSumOfSquares = (number) => {
+    let result;
+    sumOfSquares = false;
+    for (i = 0; i <= number; i++) {
+        for (k = 0; k <= number; k++) {
```
Comment (kuism) [inline 014_number_as_sum_of_squares.js]: k not defined

File: 014_number_as_sum_of_squares.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            result = (i * i + k * k);
+            if (result === number) {
+                sumOfSquares = true;
```
Comment (kuism) [inline 014_number_as_sum_of_squares.js]: Can you add the output as pull request description?

Comment (akhil-kk) [reply 014_number_as_sum_of_squares.js]: ok.

Comment (akhil-kk) [reply 014_number_as_sum_of_squares.js]: will update.

Comment (akhil-kk) [reply 014_number_as_sum_of_squares.js]: I have added the output in the pull request description.

**Review by kuism — APPROVED** [review]

**Pull request merged**